x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/index.ts

```diff
@@ -104,7 +104,7 @@ export function createApmEventClient({
 
       nextParams.body.query.bool.filter.push({
         terms: {
-          [PROCESSOR_EVENT]: apm.events,
+          [PROCESSOR_EVENT]: uniq(apm.events),
```

Comment:
When is `uniq` needed here? Is it likely that a we'll pass `events: ['transaction', 'transaction']` by mistake?